presto-geospatial/src/main/java/com/facebook/presto/geospatial/BingTile.java

```diff
@@ -70,7 +72,8 @@ public String toString()
             .toString();
     }
 
-    public static BingTile fromCoordinates(int x, int y, int zoomLevel)
+    @JsonCreator
+    public static BingTile fromCoordinates(@JsonProperty("x") int x, @JsonProperty("y") int y, @JsonProperty("zoom") int zoomLevel)
```

Comment:
Put the arguments on separate lines since the annotations makes them long (we always do that for JSON constructors)